core/objectaction/object.go

```diff
@@ -52,6 +53,9 @@ type (
 func New(opts ...funcopt.O) *T {
 	t := &T{}
 	_ = funcopt.Apply(t, opts...)
+	if t.NodeSelector != "" && t.DefaultOutput == "" {
```

Comment:
same default output regardless node selector value ?
This may help client to diagnose from output